engine/src/em-clipboard.cpp

```diff
@@ -0,0 +1,111 @@
+/* Copyright (C) 2015 LiveCode Ltd.
+ 
+ This file is part of LiveCode.
+ 
+ LiveCode is free software; you can redistribute it and/or modify it under
+ the terms of the GNU General Public License v3 as published by the Free
+ Software Foundation.
+ 
+ LiveCode is distributed in the hope that it will be useful, but WITHOUT ANY
+ WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ for more details.
+ 
+ You should have received a copy of the GNU General Public License
+ along with LiveCode.  If not see <http://www.gnu.org/licenses/>.  */
+
+
+
+#include "em-clipboard.h"
+
+
+MCRawClipboard* MCRawClipboard::CreateSystemClipboard()
+{
+    return new MCEmscriptenRawClipboard;
+}
+
+MCRawClipboard* MCRawClipboard::CreateSystemSelectionClipboard()
+{
+    return new MCEmscriptenRawClipboard;
+}
+
+MCRawClipboard* MCRawClipboard::CreateSystemDragboard()
+{
+    return new MCEmscriptenRawClipboard;
+}
+
+
+uindex_t MCEmscriptenRawClipboard::GetItemCount() const
+{
+    return 0;
+}
+
+const MCRawClipboardItem* MCEmscriptenRawClipboard::GetItemAtIndex(uindex_t p_index) const
+{
+    return NULL;
+}
+
+MCRawClipboardItem* MCEmscriptenRawClipboard::GetItemAtIndex(uindex_t p_index)
+{
+    return NULL;
+}
+
+void MCEmscriptenRawClipboard::Clear()
+{
+    ;
```

Comment:
Do you need the empty statement here, or could you just define the function body as `{}`?